MapboxSearch/sample/src/main/java/com/mapbox/search/sample/MainActivity.kt

```diff
@@ -142,6 +145,21 @@ class MainActivity : AppCompatActivity() {
             })
         }
 
+        mapView.mapboxMap.addOnMapClickListener { point ->
+            val screenCoords = mapView.mapboxMap.pixelForCoordinate(point)
+
+            mapView.mapboxMap.queryRenderedFeatures(
+                RenderedQueryGeometry(screenCoords),
+                RenderedQueryOptions(listOf("poi-label"), null)
```

Comment:
I just want to highlight that in MapsSDK v11 and our default Standard style this layer is "private", so this code won't work. It will work for other styles though.

Reply:
Will create a subsequent pull request that switches the map styles.